archimedes/archimedes/functions/utils/json_utils.py

```diff
@@ -0,0 +1,5 @@
+import json
+
+
+def json_loads(json_string):
+    return json.loads(json_string)
```

Comment:
Does `import_json` from dataflow/path.py not work?

Reply:
Oh, thanks! I'll update.

Reply:
oops, *input, but you probably figured that out already lol